schema.sql

```diff
@@ -0,0 +1,9 @@
+/* Database schema to keep the structure of entire database. */
+
+create table animals(id INT PRIMARY KEY NOT NULL, 
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to make the `id` column autoincremented so that you don't have to keep track of the next `id` your data should have. This will be taken care of automatically for you. See suggestion below;

```suggestion

create table animals(id INT PRIMARY KEY NOT NULL GENERATED ALWAYS AS IDENTITY,
```

Reply:
@julie-ify , ok thank you for the advice.